internal/server/handler_v1.go

```diff
@@ -532,20 +532,23 @@ func (s *Server) SearchStatVar(
 	if err != nil {
 		return nil, err
 	}
-	if len(localResp.StatVars) == 0 && len(localResp.Matches) == 0 {
-		if s.metadata.RemoteMixerDomain != "" {
-			remoteResp := &pb.SearchStatVarResponse{}
-			if err := util.FetchRemote(
-				s.metadata,
-				s.httpClient,
-				"/v1/variable/search",
-				in,
-				remoteResp,
-			); err != nil {
-				return nil, err
-			}
-			return remoteResp, nil
+
+	remoteResp := &pb.SearchStatVarResponse{}
+	if s.metadata.RemoteMixerDomain != "" {
+		if err := util.FetchRemote(
+			s.metadata,
+			s.httpClient,
+			"/v1/variable/search",
+			in,
+			remoteResp,
+		); err != nil {
+			return nil, err
 		}
 	}
-	return localResp, nil
+
+	resp := &pb.SearchStatVarResponse{
```

Comment:
Couple of suggestions for best practices. OK to implement them in a separate PR:

1. Consider moving the merge logic to `merger.go` ([example](https://github.com/datacommonsorg/mixer/blob/08a8e50c6c5c935159ea8e4a6986408d1cb4c66f/internal/merger/merger.go#L332))
2. Consider adding tests in `merger_test.go` ([example](https://github.com/datacommonsorg/mixer/blob/08a8e50c6c5c935159ea8e4a6986408d1cb4c66f/internal/merger/merger_test.go#L798)) and / or `handler_v1_test.go` ([example](https://github.com/datacommonsorg/mixer/blob/08a8e50c6c5c935159ea8e4a6986408d1cb4c66f/internal/server/handler_v1_test.go#L30))

Reply:
Done.